### ISSUE TITLE
Bad results on sparse-view CT with metal artifacts

### ISSUE THREAD

**Issue by syf0518** [Vandermode/TFPnP#9]
Thanks for your contribution, it is wonderful.I wanted to use your code for the reconstruction of sparse-view CT with metal artifacts on deeplesion dataset, but the results are not good.At the 800th step of training, psnr is still lower than 20dB.And the psnr increases by no more than 1dB after many steps of training, and sometimes decreases.I want to know if I should adjust the hyperparameter or learning rate.I would appreciate it if you could tell me.

**Comment by Vandermode** [comment]
have you ever tried manually tweak the hyperparameters of the Plug-and-play solver to see its performance instead of using the RL policy?

**Comment by syf0518** [comment]
> have you ever tried manually tweak the hyperparameters of the Plug-and-play solver to see its performance instead of using the RL policy?

No, I haven't tried that.What are the hyperparameters of the Plug-and-play solver in options.py?

**Comment by Vandermode** [comment]
well, it might be a case, PnP algorithm with a generic denoiser trained on natural datasets doesn't work well in your CT dataset. In this situation, probably a training-based approach such as deep unrolling would achieve better results. See our latest work
[Delta-Prox](https://github.com/princeton-computational-imaging/Delta-Prox) for details. It should be easy to realize the CT application in our domain-specific language. Let me know if you encounter any problem:)
